db/data.py

```diff
@@ -92,6 +95,47 @@ def update_spot(spot_id, spotName, spotAddress, spotCapacity, spotImage):
     return response
 
 
+def update_individual_spot_factor(spot_id, factorName, factorRating):
+    if not (1 <= factorRating <= 10):
+        return
+    spotFactor = dbc.get_spot_factor(spot_id, factorName)
+    today = datetime.today().date()
+
+    spot_document = {}
+    spot_document["spotUpdate"] = str(datetime.datetime.now())
+    
+    # if dates are not the same, then new day has begun, need to reset data with current input
+    if datetime.strptime(spotFactor["factorDate"], '%Y-%m-%d').date() != today:
+        print("Wrong date")
```

Comment:
I think maybe `print("Resetting factors with today's data: %s", date )` would be a more helpful log

Reply:
good call!